packages/server/src/types/accounts-server-options.ts

```diff
@@ -0,0 +1,33 @@
+import { User, DatabaseInterface } from '@accounts/types';
+import { EmailTemplateType } from './email-template-type';
+import { EmailTemplatesType } from './email-templates-type';
+import { UserObjectSanitizerFunction } from './user-object-sanitizer-function';
+import { ResumeSessionValidator } from './resume-session-validator';
+import { PrepareMailFunction } from './prepare-mail-function';
+import { SendMailType } from './send-mail-type';
+
+export interface AccountsServerOptions {
+  db: DatabaseInterface;
+  tokenSecret: string;
+  tokenConfigs?: {
+    accessToken?: {
+      expiresIn?: string;
+    };
+    refreshToken?: {
+      expiresIn?: string;
+    };
+  };
+  emailTokensExpiry?: number;
+  emailTemplates?: EmailTemplatesType;
+  userObjectSanitizer?: UserObjectSanitizerFunction;
+  impersonationAuthorize?: (
+    user: User,
+    impersonateToUser: User
+  ) => Promise<any>;
+  resumeSessionValidator?: ResumeSessionValidator;
+  siteUrl?: string;
+  prepareMail?: PrepareMailFunction;
+  sendMail?: SendMailType;
+  // https://github.com/eleith/emailjs#emailserverconnectoptions
```

Comment:
I think we can remove this line no? The email option is not needed anymore since we now use the sendMail property.

Reply:
I removed it :)

Reply:
Can you also remove the comment ? ^^

Reply:
done !